src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/ForAll.cs

```diff
@@ -74,6 +76,63 @@ public override string GetUniqueTexlRuntimeName(bool isPrefetching = false)
             return GetUniqueTexlRuntimeName(suffix: isPrefetching ? "_ParallelPrefetching" : string.Empty);
         }
     }
+
+    internal sealed class ForAllFunction_UO : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool SupportsParamCoercion => false;
+
+        public ForAllFunction_UO()
+            : base(ForAllFunction.ForAllInvariantFunctionName, TexlStrings.AboutForAll, FunctionCategories.Table, DType.Unknown, 0x2, 2, 2, DType.UntypedObject)
+        {
+            ScopeInfo = new FunctionScopeInfo(this);
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.ForAllArg1, TexlStrings.ForAllArg2 };
+        }
+
+        public override bool CheckInvocation(TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            Contracts.AssertValue(args);
+            Contracts.AssertAllValues(args);
+            Contracts.AssertValue(argTypes);
```

Comment:
>AssertValue [](http://example.com/codeflow?start=22&length=11)

Contracts.AssertAllValues(argTypes);

Reply:
Added

Reply:
I think you just added in above function. It needs to be added here as well.

Reply:
Fixed